apps/dashboard/src/pages/workflows.tsx

```diff
@@ -1,5 +1,10 @@
 import { WorkflowList } from '@/components/workflow-list';
+import { DashboardLayout } from '@/components/dashboard-layout';
 
 export const WorkflowsPage = () => {
-  return <WorkflowList />;
+  return (
+    <DashboardLayout headerStartItems={<h1 className="text-foreground-950">Workflows</h1>}>
+      <WorkflowList />
+    </DashboardLayout>
```

Comment:
This is how we will use the DashboardLayout on all the pages.

Reply:
This means that it is going to be remounted. Can't we specify it on each page when we define the routes?

Reply:
Not sure if that means that it won't but just asking